coursedata/level-defaults/nb_NO.yaml

```diff
@@ -1,354 +1,354 @@
 6:
     commands:
     -   name: '{print}'
-        explanation: print exactly using quotation marks
-        example: "Example: {print} '5 times 5 is ' 5 * 5"
-        demo_code: "{print} '5 times 5 is ' 5 * 5"
+        explanation: skriv ut nøyaktig det du vil med fnutter
+        example: "Eksempel: {print} '5 {times} 5 er ' 5 * 5"
+        demo_code: "{print} '5 {times] 5 er ' 5 * 5"
```

Comment:
`times` in this case should **not** have placeholder curly brackets as it is used as a normale English sentence and not as a keyword! This is wrong and could potentially make weird sentences in our examples. Also the (faulty) placeholder on line 6 `{times]` crashes our tests because the curly bracket is never closed.

@Mark-Giesen what would be the best way to fix this? Through Weblate or can I commit on this branch?

Reply:
I just fixed this through Weblate by replacing `{times]` with `ganget`. Not sure if this is the right translation (as I don't speak Norsk), but it will at least solve our current error. @Kegulf Feel free to alter the translation on Weblate again if my fix is incorrect!

Reply:
`ganger` 😁 Close one though 🧡 

You can say `5 ganger 5` or `5 ganget med 5`

Reply:
I will fix 😊